samples/bluetooth/mesh/light/src/main.c

```diff
@@ -23,8 +23,17 @@ static void bt_ready(int err)
 
 	printk("Bluetooth initialized\n");
 
-	dk_leds_init();
-	dk_buttons_init(NULL);
+	err = dk_leds_init();
```

Comment:
Could you please apply this change for all samples (and in a separate commit)? **Yes, I know**, this is not part of the task, **but** since you are touching this part, it would be nice to have the same code in all samples. It will also help others if they also meet this problem with LEDs and buttons.